### PR TITLE
Add net.audiorelay.AudioRelay

### PR DESCRIPTION
**Pull request by IverCoder**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link: [AudioRelay community](https://community.audiorelay.net/t/experimental-unofficial-flatpak-build-of-audiorelay)**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.): No patches were used.*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: net.audiorelay.AudioRelay.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <description>
+    <p>
+      Turn your phone into a microphone or speakers for PC. Use Wi-Fi or USB to share audio to multiple devices with low delay.
+    </p>
+  </description>
+    
+  <launchable type="desktop-id">net.audiorelay.AudioRelay.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://imgur.com/9o0GqbK.png</image>
```
Comment (hfiguiere) [inline net.audiorelay.AudioRelay.metainfo.xml:27]: imgur is not a reliable location for screenshots.

File: net.audiorelay.AudioRelay.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - --device=dri
+  - --persist=.skiko
+  - --persist=.java
+
+modules:
+  - name: audiorelay
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/audiorelay
+      - tar -xf audiorelay.tar.gz -C /app/audiorelay
```
Comment (hfiguiere) [inline net.audiorelay.AudioRelay.yml:21]: I don't see a JDK installed.

File: net.audiorelay.AudioRelay.yml
Comment (IverCoder) [inline net.audiorelay.AudioRelay.yml:1]: Unfortunately, the app crashes when given the Wayland permission :(

File: net.audiorelay.AudioRelay.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      - type: file
+        path: audiorelay
+      - type: file
+        path: net.audiorelay.AudioRelay.desktop
+      - type: file
+        path: net.audiorelay.AudioRelay.metainfo.xml
+      - type: file
+        dest-filename: audiorelay.tar.gz
+        only-arches:
+          - x86_64
```
Comment (hfiguiere) [inline net.audiorelay.AudioRelay.yml:36]: this isn't enough to restrict the build to x86_64.

File: net.audiorelay.AudioRelay.yml (hunk 2 above)
Comment (IverCoder) [reply net.audiorelay.AudioRelay.yml:21]: The app already bundles the Java in its archive, I think it's supposed to be simply unzipped as intended, though please correct me if I'm wrong.

File: net.audiorelay.AudioRelay.yml (hunk 3 above)
Comment (IverCoder) [reply net.audiorelay.AudioRelay.yml:36]: I have added the necessary things in `flathub.json`.

File: net.audiorelay.AudioRelay.metainfo.xml (hunk 1 above)
Comment (IverCoder) [reply net.audiorelay.AudioRelay.metainfo.xml]: I have updated the link to the app's official docs.

**Comment by bbhtt** [comment]
bot, build net.audiorelay.AudioRelay

File: net.audiorelay.AudioRelay.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.audiorelay.AudioRelay</id>
+  
+  <name>AudioRelay</name>
+  <summary>Stream audio between your devices</summary>
+  <developer_name>Asapha Halifa</developer_name>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```
Comment (bbhtt) [inline net.audiorelay.AudioRelay.metainfo.xml:10]: Is this redistributable? If there are issues with ToS, it should use extra-data I think.

Comment (IverCoder) [reply net.audiorelay.AudioRelay.metainfo.xml:10]: I think I can assume we can redistribute that, they don't even have any ToS on their website or in the app.

Comment (hfiguiere) [reply net.audiorelay.AudioRelay.metainfo.xml:10]: > I think I can assume we can redistribute that

Actually, unless specifically stated, we can assume it cannot be redistributed.

**Pull request closed without merging**